plotly/figure_factory/_trisurf.py

```diff
@@ -160,7 +160,7 @@ def trisurf(x, y, z, simplices, show_colorbar, edges_color, scale,
                 color=[min_mean_dists, max_mean_dists],
                 colorscale=colorscale,
                 showscale=True),
-            hoverinfo='None',
+            hoverinfo='none',
```

Comment:
Why was this change made?
On another note, the fact that this was `'None'` before seems a little odd to me: why not just `None`?

Reply:
Context: here is the schema for `scatter3d.hoverinfo`
```
            "hoverinfo": {
                    "arrayOk": true,
                    "description": "Determines which trace information appear on hover. If `none` or `skip` are set, no information is displayed upon hovering. But, if `none` is set, click and hover events are still fired.",
                    "dflt": "all",
                    "editType": "calc",
                    "extras": [
                        "all",
                        "none",
                        "skip"
                    ],
                    "flags": [
                        "x",
                        "y",
                        "z",
                        "text",
                        "name"
                    ],
                    "role": "info",
                    "valType": "flaglist"
                },
```

The string `'none'` is one of the valid `extras` states. Currently, I require all `flaglist` values to be a case-sensitive match to those in the schema.  Setting `hoverinfo` to the Python `None` would mean to unset any previously set value and let plotly.js choose the default (`all` in this case).

Should we be more generous and coerce case-insensitive matches to the form in the schema?

Reply:
Ah I see. I wonder if plotly.js accepted `None` before. I don't think that we should coerce the case-insensitive matches, this seems good to me.

Nice to see that this new validation is improving our own code 😉